WarehouseManagement.Api/Controllers/ZoneController.cs

```diff
@@ -86,5 +88,39 @@ public async Task<IActionResult> AllDeleted()
 
             return Ok(model);
         }
+
+        [HttpGet("entries")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        public async Task<IActionResult> Entries(int zoneId, [FromQuery] string[] statuses)
```

Comment:
we would also need to return all entries in zones, but i guess that would be into its own controller?

Reply:
You mean every single entry with its zone? If so, then yes, it should be done in EntryController in action called "All".